core/models/asset.py

```diff
@@ -301,6 +302,42 @@ class Meta:
         ordering = ['-id']
 
 
+class AssetAssignee(models.Model):
+    department = models.OneToOneField('Department', null=True, blank=True,
+                                      on_delete=models.CASCADE)
+    user = models.OneToOneField('User', null=True, blank=True,
+                                on_delete=models.CASCADE)
+
+    def __str__(self):
+        if self.department:
+            return str(self.department)
+        if self.user:
+            return str(self.user)
+        else:
+            return "No Assignee Here"
+
+    @property
+    def first_name(self):
+        if self.department:
+            return self.department.name
+        if self.user:
+            return self.user.first_name
+
+    @property
+    def last_name(self):
+        if self.department:
+            return self.department.name
+        if self.user:
+            return self.user.last_name
+
+    @property
+    def email(self):
+        if self.department:
+            return self.department.name
```

Comment:
**Note to self**: Department email?

<img width="1652" alt="screen shot 2018-08-14 at 13 13 32" src="https://user-images.githubusercontent.com/10899666/44086261-aabfdb48-9fc4-11e8-912a-04209cd8a7e2.png">

Reply:
on this, what would you suggest? Should we add an email field to a department or return an empty string??

Reply:
Leave it as is for now.